test/performance/shmem_perf_suite/bw_common.h

```diff
@@ -127,10 +137,10 @@ int static inline partner_node(perf_metrics_t my_info)
         return 0;
 
     if(my_info.cstyle == COMM_PAIRWISE) {
-        int pairs = my_info.num_pes / 2;
+        int pairs = my_info.midpt;
 
-        return (my_info.my_node < pairs ? my_info.my_node + pairs :
-                    my_info.my_node - pairs);
+        return (my_info.my_node < pairs ? (my_info.my_node + pairs) :
+            (my_info.my_node - pairs));
```

Comment:
If ```num_pes``` is odd, the ```num_pes-1``` PE is an "extra PE" and do the wrong thing here.  Or is this problem already elimiated earlier on in the benchmark?